rmgpy/rmg/modelTest.py

```diff
@@ -118,7 +120,13 @@ def setUpClass(cls):
         for family in rmg.database.kinetics.families.values():
             family.forbidden = ForbiddenStructures()
         rmg.database.forbiddenStructures = ForbiddenStructures()
-
+        
+        rmg.database.thermo = ThermoDatabase()
+        database = ThermoDatabase()
+        libraries = ['primaryThermoLibrary', 'GRI-Mech3.0']
+        path = os.path.join(settings['database.directory'], 'thermo')
```

Comment:
If you want to use a database, which isn't typically recommended unless testing the database, you should use the testing database, not the main database. Otherwise, changes to primaryThermoLibrary might make it appear that Thermo pruning no longer works. I think it is better just to manually input thermo in this file for the tests you are interested in to avoid any issues with later database changes (main or test).